src/services/dependencies.service.js

```diff
@@ -18,6 +18,23 @@ export const installDependency = (
   });
 };
 
+export const installDependencies = (
+  projectPath: string,
+  dependencies: Array<{ dependencyName: string, version: string }>
+) => {
+  return new Promise((resolve, reject) => {
+    childProcess.exec(
+      `npm install ${dependencies
+        .map(({ dependencyName, version }) => `${dependencyName}@${version}`)
+        .join(' ')} -SE`,
+      { cwd: projectPath },
+      (err, res) => {
+        err ? reject(err) : resolve(res);
+      }
+    );
+  });
+};
```

Comment:
I think we should update `installDependency` to delegate to this method, to DRY it up a bit.

```js
const installDependency = (projectPath, dependencyName, version) => (
  installDependencies(projectPath, [{ dependencyName, version }])
)
```

I believe the resolve/rejected values would be identical, so this should work I think

Reply:
This could be a nice way to make the action flow I outlined in one of my first comments more intuitive. Internally, all calls would be routed to `installDependencies`, but there would be distinct action creators for singular and plural installs. Does that make sense, or should we just stick to plural action names?

```
INSTALL_DEPENDENCY_START (singular)
INSTALL_DEPENDENCY_ERROR
INSTALL_DEPENDENCY_FINISH
INSTALL_DEPENDENCIES_START (plural)
INSTALL_DEPENDNECIES_ERROR
INSTALL_DEPENDENCIES_FINISH
```

OR

```
INSTALL_DEPENDENCIES_START (singular and plural)
INSTALL_DEPENDENCIES_ERROR
INSTALL_DEPENDENCIES_FINISH
```

Reply:
I'm cool with consolidating it to always be plural! 

Actually I'm happy either way (so long as action-creators and actions are always mapped to each other; I wouldn't like for `installDependencyStart` to have type `INSTALL_DEPENDENCIES_START`, or vice versa).